_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -120,12 +120,56 @@ just be more laid back.-->{% endcomment %}
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jonatack
+[Periodically make block-relay connections and sync headers][review club
+#19858] is a PR ([#19858][Bitcoin Core #19858]) by Suhas Daftuar to make eclipse
+attacks more difficult by increasing the existing outbound peer rotation beyond
+the current logic of stale tip detection and replacement with a new full-relay
+connection. The proposal initiates outbound connections every 5 minutes on
+average to sync headers and potentially learn of new blocks, in which case it
```

Comment:
The first time I read that, I read it as always switching out a peer when it initiates a new outbound connection every five minutes. 

How about: "The proposal initiates outbound connections every 5 minutes on average to sync headers with a new peer. When it learns about new blocks, it rotates out an existing block-relay-only peer in favor of this new peer."